tasks/options/webpack.js

```diff
@@ -1,31 +1,9 @@
 /* eslint-disable comma-dangle */
 
-var webpack = require('webpack');
-
 module.exports = function(grunt) {
-    var config = require('../../webpack.config.js')(grunt);
-
-    config.progress = !grunt.option('webpack-no-progress');
-    config.devtool = grunt.option('webpack-devtool') || 'source-map';
-
-    config.module.rules = config.module.rules.map((rule) => {
-        if (rule.loader === 'ts-loader') {
-            // skipping typechecking here to make build faster
-            // types are checked in `npm run lint` using
-            // typescript compiler directly which is faster
-            rule.options.transpileOnly = true;
```

Comment:
dropping `transpileOnly` might make webpack dev server reloads slower.

Reply:
it's set in webpack.config.js